concepts/generators/about.md

```diff
@@ -1,2 +1,132 @@
-#TODO: Add about for this concept.
+# About
 
+## Constructing a generator
+
+Generators are constructed much like other looping or recursive functions, but require a [`yield` expression](#the-yield-expression), which we will explore in depth a bit later.
+
+
+An example is a function that returns the _squares_ from a given list of numbers.  
+As currently written, all input must be processed before any values can be returned:  
+
+
+```python
+>>> def squares(list_of_numbers):
+>>>     squares = []
+>>>     for number in list_of_numbers:
+>>>         squares.append(number ** 2)
+>>>     return squares
+```
+
+You can convert that function into a generator like this:
+
+```python
+def squares(list_of_numbers):
+    for number in list_of_number:
+        yield number ** 2
+```
+
+The rationale behind this is that you use a generator when you do not need all the values _at once_.
+
+This saves memory and processing power, since only the value you are _currently working on_ is calculated.
+
+
+## Using a generator
+
+Generators may be used in place of most `iterables` in Python. This includes  _functions_ or _objects_ that require an `iterable`/`iterator` as an argument.
+
+To use the `squares()` generator:
+
+```python
+>>> squared_numbers = squares([1, 2, 3, 4])
+
+>>> for square in squared_numbers:
+>>>     print(square)
+1
+4
+9
+16
+```
+
+Values within a generator can also be produced/accessed via the `next()` function. 
+`next()` calls the `__next__()` method of a generator object, "advancing" or evaluating the generator code up to its `yield` expression, which then "yields" or returns the value.
+
+```python
+square_generator = squares([1, 2])
+
+>>> next(square_generator)
+1
+>>> next(square_generator)
+4
+```
+
+When a `generator` is fully consumed and has no more values to return, it throws a `StopIteration` error.
+
+```python
+>>> next(square_generator)
+Traceback (most recent call last):
+  File "<stdin>", line 1, in <module>
+StopIteration
+```
+
+### Difference between iterables and generators
+
+Generators are a special sub-set of _iterators_.
+`Iterators` are the mechanism/protocol that enables looping over _iterables_.
+Generators and and the iterators returned by common Python (`iterables`)[https://wiki.python.org/moin/Iterator] act very similarly, but there are some important differences to note:

```

Comment:
```suggestion
Generators and the iterators returned by common Python (`iterables`)[https://wiki.python.org/moin/Iterator] act very similar, but there are some important differences to note:

```

The double "and" for sure.  I think that "similar" is more correct given the phrase.